pkg/controllers/dashboardapi/settings/settings_test.go

```diff
@@ -24,34 +24,40 @@ type testCase struct {
 	existingSetting *v3.Setting
 }
 
-var settingsETCD = make(map[string]*v3.Setting)
-
 func TestSetAll(t *testing.T) {
```

Comment:
Nit: Might be good to assert (in some way) that the settings which were in the map don't have the unknown label on them (maybe in teh asserts on the test cases, also assert that finalSetting.Labels doesn't contain the label).

Reply:
Agreed.